airbyte-config/init/src/main/resources/seed/destination_definitions.yaml

```diff
@@ -189,21 +225,11 @@
   documentationUrl: https://docs.airbyte.io/integrations/destinations/snowflake
   icon: snowflake.svg
   resourceRequirements:
-    # this is our first example of setting these requirements. they are guesses, not data driven.
-    # setting default cpu higher, because we have found that check and discover can be cpu constrained for dbs.
-    default:
-      cpu_limit: "1.0"
-      cpu_request: "1.0"
-      memory_limit: "300Mi"
-      memory_request: "300Mi"
```

Comment:
Removing this because these resource requirements are not currently applied to anything besides sync jobs, so having this here is a bit misleading. I created an issue to add support for this here: https://github.com/airbytehq/airbyte/issues/10862